build_tools/kickstarts/centos-7-kubernetes-vagrant.ks

```diff
@@ -93,6 +93,15 @@ tuned-adm profile virtual-guest
 echo "%vagrant ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers.d/vagrant
 sed -i "s/^.*requiretty/#Defaults requiretty/" /etc/sudoers
 
+# Fix for #128
```

Comment:
@praveenkumar lets put the complete link as the issue. It would be easier to track it later i.e. s/#128/https://github.com/projectatomic/adb-atomic-developer-bundle/issues/128/